README.md

```diff
@@ -1,8 +1,13 @@
-ELK autopilot pattern
+Autopilot Pattern ELK
 ==========
 
 *[Autopilot pattern](http://autopilotpattern.io/) implementation of ELK*
 
+[![DockerPulls](https://img.shields.io/docker/pulls/autopilotpattern/mysql.svg)](https://registry.hub.docker.com/u/autopilotpattern/mysql/)
```

Comment:
Thanks.

Fixed, but now it looks like this image isn't pushed to the autopilotpattern org on Docker Hub.